libevm/JitVM.cpp

```diff
@@ -175,9 +174,27 @@ void create(evm_result* o_result, ExtVMFace& _env, evm_message const* _msg) noex
 	}
 	else
 	{
-		o_result->code = EVM_FAILURE;
-		o_result->output_data = nullptr;
-		o_result->output_size = 0;
+		o_result->code = EVM_REVERT;
```

Comment:
One example of future use: `EVM_STACK_UNSERFLOW`, output: `"Stack overflow in instruction ADD at position 321, instruction counter: 12344"`.

Receiving this EVM can log it, but should not copy it to the RETURNDATA buffer.